docs/migration/migrate_8_0.asciidoc

```diff
@@ -80,6 +80,158 @@ logging:
 -------------------
 See https://github.com/elastic/kibana/pull/87939 for more details.
 
+[float]
+==== Logging destination is specified by the appender
+*Details:* Previously log destination would be `stdout` and could be changed to `file` using `logging.dest`. 
+
+*Impact:* To restore the previous behavior, in `kibana.yml` use the `console` appender to send logs to `stdout`.
+[source,yaml]
+-------------------
+logging:
+  loggers:
+    - name: plugins.myPlugin
+      appenders: [console]
+-------------------
+
+To send logs to `file` with a given file path, you should define a custom appender with `type:file`:
+[source,yaml]
+-------------------
+logging:
+  appenders:
+    file:
+      type: file
+      fileName: /var/log/kibana.log
+      layout:
+        type: pattern
+  loggers:
+    - name: plugins.myPlugin
+      appenders: [file]
+-------------------
+
+[float]
+==== Specify log event output with root
+*Details:* Previously logging output would be specified by `logging.silent` (none), 'logging.quiet' (error messages only) and `logging.verbose` (all).
+
+*Impact:* To restore the previous behavior, in `kibana.yml` specify `logging.root.level` as one of `off`, `error`, `all`:
+[source,yaml]
+-------------------
+# suppress all logs
+logging:
+  root:
+    appenders: [default]
```

Comment:
@lukeelmers At the moment we get a config error if we try to run Kibana without the default appender declared for the root logger. While it makes sense to _not_ include it in the `migrate_8_0` guide, I'm concerned that some folks might get tripped up if they configure Kibana without the default appender and it's still required.

Reply:
> While it makes sense to not include it in the migrate_8_0 guide, I'm concerned that some folks might get tripped up if they configure Kibana without the default appender and it's still required.

Fair enough, I suppose if the point of the 8.0 migration guide is to allow users to make changes _before_ they hit 8.0, keeping the extra `default` appender in the example makes it easier, even if it won't be strictly necessary in 8.0.

Reply:
> At the moment we get a config error if we try to run Kibana without the default appender declared for the root logger.

Do we know why this is necessary? Is it possible to relax this requirement now that all log messages (including requests) are going through the KP logger? If we can relax this requirement, we can do that in a separate PR.

Reply:
@joshdover That's a good point, I believe the intent was to make sure that nobody can accidentally turn off the legacy logs and lose log data. But as you point out, we should now have feature parity with legacy logs so it might be okay to allow folks to remove this before 8.0, if they choose to do so.

Reply:
@TinaHeiligers did we capture this anywhere? May be worth discussing prioritizing this during our weekly

Reply:
@joshdover I'll add it to the sync agenda